--- Geant4PackageCache.cmake	2023-12-27 23:01:57.562748472 +0800
+++ Geant4PackageCache.cmake-new	2023-12-27 23:01:42.961529575 +0800
@@ -19,7 +19,7 @@
 
 # EXPAT Build Time Settings
 geant4_set_and_check_package_variable(EXPAT_INCLUDE_DIR "/usr/include" PATH "Path to a file.")
-geant4_set_and_check_package_variable(EXPAT_LIBRARY ""  "")
+#geant4_set_and_check_package_variable(EXPAT_LIBRARY ""  "")
 
 # ZLIB Build Time Settings
 geant4_set_and_check_package_variable(ZLIB_INCLUDE_DIR "/usr/include" PATH "Path to a file.")
